force-app/main/default/classes/SOQL.cls

```diff
@@ -50,9 +52,18 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static List<Id> getContactIdsByLastName() {
     //  The body of the method
-    //}
+    List<Contact> contacts = [SELECT Id FROM Contact WHERE LastName = 'Smith' OR LastName = 'Jones'];
+
+        List<Id> contactIds = new List<Id>();
+            for (Contact contact : contacts) {
+            
+                contactIds.add(contact.Id);
+    }
+    
+    return contactIds;
```

Comment:
You can also query right into a Map new Map<Id, Contact>([..query..]) and then return the keySet of that map. Saves you from doing the looping yourself.